app/src/main/java/com/vultisig/wallet/ui/screens/ScanQrScreen.kt

```diff
@@ -78,45 +96,111 @@ internal fun ScanQrScreen(
                 .addOnSuccessListener(onSuccess)
         }
     }
+    val textColor = MaterialTheme.colorScheme.onBackground
+    val appColor = Theme.colors
+    val dimens = MaterialTheme.dimens
 
-    UiBarContainer(
-        navController = navController,
-        title = stringResource(R.string.scan_qr_default_title),
-        endIcon = R.drawable.ic_gallery,
-        onEndIconClick = {
-            pickMedia.launch(PickVisualMediaRequest(PickVisualMedia.ImageOnly))
-        }
-    ) {
-        if (cameraPermissionState.status.isGranted) {
-            QrCameraScreen(
-                onSuccess = onSuccess,
-            )
-        } else if (cameraPermissionState.status.shouldShowRationale) {
-            Box(
-                modifier = Modifier.fillMaxSize()
-            ) {
-                Text(
-                    text = stringResource(R.string.camera_permission_denied),
-                    textAlign = TextAlign.Center,
-                    color = Theme.colors.neutral100,
-                    style = Theme.montserrat.titleMedium,
-                    modifier = Modifier.align(Alignment.Center),
+    Scaffold(
+        bottomBar = {
+            if (cameraPermissionState.status.isGranted.not())
+                MultiColorButton(
+                    minHeight = dimens.minHeightButton,
+                    backgroundColor = appColor.turquoise800,
+                    textColor = appColor.oxfordBlue800,
+                    iconColor = appColor.turquoise800,
+                    textStyle = Theme.montserrat.titleLarge,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(
+                            start = dimens.marginMedium,
+                            end = dimens.marginMedium,
+                            bottom = dimens.marginMedium,
+                        ),
```

Comment:
Please, refrain from using dimens; they're breaking the layout. Just specify values from Figma.